File: backend/src/main/java/de/riedeldev/sunplugged/sputter/backend/model/CryoState.java
package de.riedeldev.sunplugged.sputter.backend.model;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class CryoState {

	private Boolean compressorStarted;

	private Boolean cryo1CompressorOn;

	private Boolean cryo2CompressorOn;

	private Double cryo1Temp;

	private Double cryo2Temp;

	private Boolean roughValveOne;

	private Boolean roughValveTwo;

	private Boolean purgeValveOne;

	private Boolean purgeValveTwo;

	private Boolean purgeHeaterOne;

	private Boolean purgeHeaterTwo;

}

File: frontend/src/app/model/cryoState.ts
export interface CryoState {
    
    compressorStarted: boolean;

    cryo1CompressorOn: boolean;

    cryo2CompressorOn: boolean;

    cryo1Temp: number;

    cryo2Temp: number;

    roughValveOne: boolean;

    roughValveTwo: boolean;

    purgeValveOne: boolean;

    purgeValveTwo: boolean;

    purgeHeaterOne: boolean;

    purgeHeaterTwo: boolean;
}